File: official/docs/python/current/parcels/create.py
import os

import easypost


client = easypost.EasyPostClient(os.getenv("EASYPOST_API_KEY"))

parcel = client.parcel.create(
    length=20.2,
    width=10.9,
    height=5,
    weight=65.9,
)

print(parcel)

File: official/docs/csharp/current/refunds/create.cs
using System;
using System.Collections.Generic;
using System.Threading.Tasks;
using Newtonsoft.Json;
using EasyPost;
using EasyPost.Models.API;
using EasyPost.Parameters;

namespace EasyPostExamples
{
    public class Examples
    {
        public static async Task Main()
        {
            var client = new EasyPost.Client(new EasyPost.ClientConfiguration("EASYPOST_API_KEY"));

            Parameters.Refund.Create parameters = new()
            {
                Carrier = "USPS",
                TrackingCodes = new List<string> { "EZ1000000001" }
            };

            List<Refund> refunds = await client.Refund.Create(parameters);

            Console.WriteLine(JsonConvert.SerializeObject(refunds, Formatting.Indented));
        }
    }
}
